src/ripple/protocol/impl/ErrorCodes.cpp

```diff
@@ -107,7 +107,8 @@ constexpr static ErrorInfo unorderedErrorInfos[]{
     {rpcSTREAM_MALFORMED,       "malformedStream",      "Stream malformed.", 400},
     {rpcTOO_BUSY,               "tooBusy",              "The server is too busy to help you now.", 503},
     {rpcTXN_NOT_FOUND,          "txnNotFound",          "Transaction not found.", 404},
-    {rpcUNKNOWN_COMMAND,        "unknownCmd",           "Unknown method.", 405}};
+    {rpcUNKNOWN_COMMAND,        "unknownCmd",           "Unknown method.", 405},
+    {rpcAMM_ISSUE_MALFORMED,    "ammIssueMalformed",    "AMM issue is malformed.", 406}};
```

Comment:
I think it's possible that some future transaction type could also use "Issue"s, so I don't think it's really necessary or helpful to tie this error to AMM. It could simply be.

```
{rpcISSUE_MALFORMED,    "issueMalformed",    "Issue is malformed.", 400}
```
Historically, it looks like new codes have been inserted alphabetically (with the exception of `rpcEXPIRED_VALIDATOR_LIST`, but I don't know why), so this would be better inserted at line 79 (with the new name).

The last field is an HTTP response code, not a counter, and is optional. I don't think [406 Not acceptable](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406) is the correct code here. Probably better to use 400, which the other "malformed" errors use.